dipy/io/gradients.py

```diff
@@ -38,7 +39,11 @@ def read_bvals_bvecs(fbvals, fbvecs):
             if isinstance(this_fname, string_types):
                 base, ext = splitext(this_fname)
                 if ext in ['.bvals', '.bval', '.bvecs', '.bvec', '.txt', '.eddy_rotated_bvecs', '']:
-                    vals.append(np.squeeze(np.loadtxt(this_fname)))
+                    with open(this_fname, 'rb') as f:
```

Comment:
I think we do not need to read the file in binary mode, `read` mode is enough so can you replace `rb` by `r`.